nsls2forge_utils/check_results.py

```diff
@@ -41,8 +45,12 @@ def check_conda_channels(forbidden_channel='conda-forge',
 
     if failed_packages:
         formatted = '\n'.join(failed_packages)
-        raise RuntimeError(f'Packages from the "{forbidden_channel}" channel '
-                           f'found:\n{formatted}')
+        if ignore_exception:
+            print(f'Packages from the "{forbidden_channel}" channel '
+                  f'found:\n{formatted}')
```

Comment:
I think the contents being printed in either case can be pulled above to a var like `msg`, and be reused here. DRY (don't repeat yourself) principle will be satisfied :)